lib/logs/log_formatter.rb

```diff
@@ -85,6 +87,16 @@ def color_background_and_style(key, value)
       when 3 then :yellow
       else %i[black red]
       end
+    when :queries
+      if value < 5
+        :green
+      elsif value < 12
+        :yellow
+      else
+        :red
+      end
+    when :cached_queries
+      :white
```

Comment:
I'm not super sure whether cached queries are good or bad. On the one hand, it seems good, since we're at least not hitting the database. But, on the other hand, should we not even be sending these queries to ActiveRecord in the first place (and, instead, we should be memoizing something at the Ruby level that we currently aren't)? So, for now, I'm going to just cop out and color this "white" (actually a bit gray).